### PR TITLE
MF-571 - Add Env.elm to set custom base URL

### PR DESCRIPTION
**Pull request by darkodraskovic**
Add Env.elm so we can set custom base URL and develop locally.

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -3,58 +3,69 @@ Dashboard made with [elm-bootstrap](http://elm-bootstrap.info/).
...
+### Native GUI build
 
-### About Elm `make`
+Install Elm (https://guide.elm-lang.org/install.html) and then edit
+*src/Env.elm* by replacing
 
-`make` does `elm make src/Main.elm`.
+```
+env =
+    { protocol = ""
```
Comment (drasko) [inline ui/README.md]: There should be some default values IMHO.

File: ui/src/Env.elm [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+-- Mainflux
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+
+module Env exposing (env)
+
+
+env =
+    { protocol = ""
```
Comment (drasko) [inline ui/src/Env.elm]: Put some default values here if needed.

I let you decide.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by nmarcetic — DISMISSED** [review]

**Review by blokovi — DISMISSED** [review]
LGTM

File: ui/README.md (hunk 1 above)
Comment (darkodraskovic) [reply ui/README.md]: No default value needed, cause we need it to work out of the box in the docker composition. We only need to input some value if it's natively deployed or user wants a special setting.

File: ui/src/Env.elm [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+-- Mainflux
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+
+module Env exposing (env)
+
+
+env =
+    { url = ""
```
Comment (drasko) [inline ui/src/Env.elm]: Please use default value. If the user did not set ENV var, it should fallback to something like `http://localhost:3000`, right?

**Review by drasko — CHANGES_REQUESTED** [review]

File: ui/README.md (hunk 1 above)
Comment (drasko) [reply ui/README.md]: Why can not it work natively with default value, and be controlled in the Docker via ENV variable?

File: ui/src/Env.elm (hunk 3 above)
Comment (drasko) [reply ui/src/Env.elm]: Probably again - defult should be left empty, because of Docker deployment?

**Review by manuio — DISMISSED** [review]

File: ui/README.md [hunk 4]
```diff
@@ -3,58 +3,52 @@ Dashboard made with [elm-bootstrap](http://elm-bootstrap.info/).
...
+`make run`
 
-Install Docker (https://docs.docker.com/install/) and Docker compose
-(https://docs.docker.com/compose/install/), `cd` to Mainflux root directory and
-then
+and follow the instructions on screen.
 
-`docker-compose -f docker/docker-compose.yml up`
+**NB:** `make` does `elm make src/Main.elm` and `make run` executes `elm
+reactor`. Cf. _Makefile_ for more options.
```
Comment (blokovi) [inline ui/README.md:39]: Meaning of Cf. abbreviation is not so common knowlegde

**Review by blokovi — DISMISSED** [review]
LGTM

File: ui/README.md (hunk 4 above)
Comment (darkodraskovic) [reply ui/README.md:39]: Let the developers learn a bit of humanities language :)

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]

**Pull request merged**